connectors/src/connectors/zendesk/temporal/workflows.ts

```diff
@@ -258,15 +285,14 @@ export async function zendeskBrandHelpCenterSyncWorkflow({
     connectorId,
     brandId,
   });
-  if (!isHelpCenterAllowed) {
-    return; // nothing to sync
+  if (isHelpCenterAllowed) {
+    await runZendeskBrandHelpCenterSyncActivities({
```

Comment:
We call 'articles' everywhere else and helpcenter here, let's try to have a consistent naming